codebuild/spec/buildspec_openssl3fips.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: 0.2
+      
+phases:
+  build:
+    on-failure: ABORT
+    commands:
+      - |
+        cmake . -Bbuild \
+          -DCMAKE_PREFIX_PATH=/usr/local/openssl-3.0-fips \
+          -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+          -DASAN=ON \
+          -DUBSAN=ON
+      - cmake --build ./build -- -j $(nproc)
+  post_build:
```

Comment:
Thank you for separating build from test... we'll need to be more pedantic about this going forward...